### PR TITLE
Changes as per Issue #196

### PR DESCRIPTION
**Pull request by rdfguy**
Explanatory paragraph added to the end of Section 2.1.

***
<a href="https://pr-preview.s3.amazonaws.com/rdfguy/rdf-star/pull/210.html" title="Last updated on Oct 4, 2021, 3:25 PM UTC (1f3220b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star/210/27e35a8...rdfguy:1f3220b.html" title="Last updated on Oct 4, 2021, 3:25 PM UTC (1f3220b)">Diff</a>

### CODE REVIEW

**Comment by gkellogg** [comment]
For a future PR, better to use a branch other than "main", which would generally mirror the "main" branch of the target repo. That way, if there is some divergence, it doesn't mess up your fork.

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
@rdfguy - the automatic IP check says "rdfguy is unknown, they need to link their GitHub account with a W3C account."

https://labs.w3.org/repo-manager/pr/id/w3c/rdf-star/210 

Would it be possible for you to add your GH account to your W3C account?

File: cg-spec/editors_draft.html [hunk 1]
```diff
@@ -455,6 +455,22 @@ <h2>Quoted and Asserted Triples</h2>
           &lt;&lt; &lt;&lt; :a :name "Alice" &gt;&gt;  :reportedBy :charlie &gt;&gt;
         </code>
       </p>
+      <p>
```
Comment (gkellogg) [inline cg-spec/editors_draft.html:458]: <div class="note">
      <p>

File: cg-spec/editors_draft.html [hunk 2]
```diff
@@ -455,6 +455,22 @@ <h2>Quoted and Asserted Triples</h2>
...
+        you can imagine that quoting prevents the expression from being
+        evaluated. In RDF (as well as in RDF-star), <em>asserting</em> a triple
+        (i.e., considering it to be &quot;true&quot;) serves as the analogue of
+        evaluation. Triples are asserted, unless (in RDF-star) they are quoted
+        which, effectively, prevents the triple from being asserted - thus a
+        quoted triple stands by itself and we assign no truth value to it.
+      </p>
+      <p>
+        Obviously this way of thinking is helpful only if you understand how Lisp works.
+      </p>
```
Comment (gkellogg) [inline cg-spec/editors_draft.html:473]: </p>
      </div>

**Review by gkellogg — CHANGES_REQUESTED** [review]
This is useful, but, even though within a non-normative section, I think it makes more sense to enclose it in a Note.

**Review by hartig — APPROVED** [review]
Fine now with the latest commit that encloses the text as a note.

**Review by gkellogg — APPROVED** [review]

**Pull request merged**